### ISSUE TITLE
Not working on offline PC

### ISSUE THREAD

**Issue by Molin-L** [AleksandarDev/vscode-sequence-diagrams#33]
Extention version: 0.4.4

Enviroment: 
Windows 10, offline;
npm 6.14.11.

# What I do
1. Installed extension using`.vsix` file downloaded from Visual Studio Code Marketplace.

2. Create a `.seqdiag` file as below:
```
A->B: Hello
```

# What I found
.1 The preview is blank.
2. Console shows:
```
DevTools failed to load SourceMap: Could not load content for https://ticino.blob.core.windows.net/sourcemaps/d06d2f1d6245ce00b1c36a9cd81a9087d225173e/node_modules/vscode-textmate/release/main.js.map: 
  ERR net::ERR_PROXY_CONNECTION_FAILED: Error: net::ERR_PROXY_CONNECTION_FAILED
    at SimpleURLLoaderWrapper.<anonymous> (electron/js2c/browser_init.js:109:6508)
    at SimpleURLLoaderWrapper.emit (events.js:315:20)
[Extension Host] vscode-sequence-diagrams: Active! (at Function.info (c:\Users\l00600327\.vscode-insiders\extensions\aleksandardev.vscode-sequence-diagrams-0.4.4\dist\extension.js:2:9834))
workbench.desktop.main.js:62 
[Extension Host] vscode-sequence-diagrams: Check refresh document rejected. (at Function.info (c:\Users\l00600327\.vscode-insiders\extensions\aleksandardev.vscode-sequence-diagrams-0.4.4\dist\extension.js:2:9834))
```

I don't know if it's because my PC is offline. I'm wondering if I miss some dependencies?

**Comment by AleksandarDev** [comment]
Possibly closed with #32 

Please see updated extension 0.4.5 :)

**Comment by Molin-L** [comment]
> Possibly closed with #32
> 
> Please see updated extension 0.4.5 :)

Thanks. 0.4.5 works totally fine.
